plugins/polio/tests/test_lqas_im_country.py

```diff
@@ -96,17 +96,3 @@ def test_list_results_filtered_by_account(self):
         response_body = self.assertJSONResponse(response, 200)
         self.assertEqual(len(response_body["results"]), 1)
         self.assertEqual(response_body["results"][0]["data"], data_store_content3)
-
-    def test_list_results_filtered_by_slug(self):
-        """GET /polio/lqasmap/country/ should only show results for lqas"""
-
-        self.client.force_authenticate(self.authorized_user_regular)
-        response = self.client.get(api_url)
-        response_body = self.assertJSONResponse(response, 200)
-        self.assertEqual(len(response_body["results"]), 2)
-        self.assertEqual(response_body["results"][0]["data"], data_store_content1)
-        self.assertEqual(response_body["results"][1]["data"], data_store_content2)
-
-        self.client.force_authenticate(self.authorized_user_regular)
-        response = self.client.get(f"{api_url}{self.data_store_im.slug}/")
-        response_body = self.assertJSONResponse(response, 404)
```

Comment:
The endpoint now only returns something if a `slug` is passed, so this test is obsolete